moonraker/components/update_manager/git_deploy.py

```diff
@@ -321,6 +322,14 @@ async def initialize(self, need_fetch: bool = True) -> None:
             self.git_owner = "?"
             if owner_match is not None:
                 self.git_owner = owner_match.group(1)
+
+            # Parse GitHub Repository Name from URL
+            repo_match = re.match(r".*/(.*)\..*", self.upstream_url)
```

Comment:
FWIW, it is possible that the upstream url won't have a ".git" extension, in which case this regex probably won't match.